src/main/java/roomescape/entity/Reservation.java

```diff
@@ -0,0 +1,60 @@
+package roomescape.entity;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import roomescape.exception.InvalidException;
+
+public class Reservation {
+
+    private final long id;
+    private final String name;
+    private final LocalDate date;
+    @JsonFormat(pattern = "HH:mm")
+    private final LocalTime time;
+
+    public Reservation(long id, String name, LocalDate date, LocalTime time) {
+        this.id = id;
+        validateName(name);
+        this.name = name;
+        validateDate(date);
+        this.date = date;
+        validateTime(time);
+        this.time = time;
+    }
```

Comment:
(제 생각에는) 일반적으로 검증 로직과 할당 로직을 뭉쳐놓는 편이에요.

```suggestion
    public Reservation(long id, String name, LocalDate date, LocalTime time) {
        validateName(name);
        validateDate(date);
        validateTime(time);
        
        this.id = id;
        this.name = name;
        this.date = date;
        this.time = time;
    }
```